app/src/main/java/io/zenandroid/onlinego/spectate/SpectateAdapter.kt

```diff
@@ -32,8 +34,10 @@ class SpectateAdapter : GamesAdapter<SpectateAdapter.ViewHolder>() {
             holder.boardView.position = pos
             holder.blackName.text = gameData.players?.black?.username
             holder.blackRank.text = formatRank(egfToRank(gameData.players?.black?.egf))
+            holder.blackRank.showIf(SettingsRepository.showRanks)
```

Comment:
FYI, `SpectateAdapter` is not used anymore, it's pretty much dead code that I keep around just in case we ever revive the feature of watching other people play. It's probably not worth our time maintaining it.